app/uk.gov.hmrc.helptosavestridefrontend/views/partials/h2.scala.html

```diff
@@ -0,0 +1,33 @@
+@*
+ * Copyright 2023 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *@
+
+@this()
+
+@(
+  key: String,
+  classes: Option[String] = None
+)(implicit
+  messages: Messages
+)
+
+@cls = @{"govuk-heading-s".concat(classes.getOrElse(""))}
```

Comment:
adding this default class means ALL h2 elements will have the small heading size applied (even if overwritten with another size) - most headings I can see in the templates use a medium size, also you might need a make sure there is a space before any concat values if done like this

Reply:
changed to medium